config/invenio.conf

```diff
@@ -2558,6 +2558,11 @@ CFG_HEPDATA_FIELD =  hepdataparent
 ## CFG_ARXIV_URL_PATTERN -- the url to use for fetching arxiv pdfs
 CFG_ARXIV_URL_PATTERN = http://export.arxiv.org/pdf/%sv%s.pdf
 
+#######################################
+## Part 37: Blacklist for Orcidutils ##
+#######################################
+
+CFG_ORCIDUTILS_BLACKLIST = []
```

Comment:
Can you use this config to just specify a relative path to a file that is read at startup? In this way we do not need to use inveniocfg --update-all and restarting Apache, everytime we want to amend the blacklist.

Additionally the blacklist should be a list of tuples: ORCID ID, DOI, because the duplicate check is performed per DOI.

Additionally would be great to better document what this black list is all about.